Code/src/main/java/nl/utwente/viskell/ui/components/Connection.java

```diff
@@ -61,20 +55,30 @@
      * @param pane The Pane this Connection is on.
      * @param anchor A ConnectionAnchor of this Connection.
      */
-    public Connection(CustomUIPane pane, ConnectionAnchor anchor) {
+    public Connection(CustomUIPane pane, OutputAnchor source, InputAnchor sink) {
```

Comment:
I like the source/sink terminology. Would it make sense to rename the anchor classes to fit?